backend/engine/plugins/github_repo_health/main.py

```diff
@@ -0,0 +1,127 @@
+import json
+
+from artemislib.github.app import GithubApp
+from engine.plugins.github_repo_health.cli.src.utilities import Checker, Config, Github
+from engine.plugins.lib import utils
+
+PLUGIN_NAME = "github_repo_health"
+
+# Will be used if service is "github", but no matching PluginConfig is found
+DEFAULT_CONFIG = {
+    "name": "artemis_default",
+    "version": "1.0.0",
+    "rules": [
+        {
+            "type": "branch_commit_signing",
+            "id": "github_branch_commit_signing",
+        },
+        {
+            "type": "branch_enforce_admins",
+            "id": "github_branch_enforce_admins",
+        },
+        {
+            "type": "branch_pull_requests",
+            "id": "github_branch_pull_requests",
+            "expect": {
+                "dismiss_stale_reviews": True,
+                "require_code_owner_reviews": True,
+            },
+            "min_approvals": 1,
+        },
+        {
+            "type": "branch_status_checks",
+            "id": "github_branch_status_checks",
+            "expect": {
+                "strict": True,
+            },
+        },
+        {
+            "type": "repo_security_alerts",
+            "id": "github_repo_security_alerts",
+        },
+        # Refer to engine/plugins/github_repo_health/cli/src/rules for other rules
+    ],
+}
+
+log = utils.setup_logging(PLUGIN_NAME)
```

Comment:
Let's use the `Logger` from `artemislib` and start phasing out this logging function.